css-animations-2/Overview.bs

```diff
@@ -845,29 +845,80 @@ intended to support using [=triggers=] for <em>other</em> purposes in the future
 For now, though, [=triggered animations=] are the only user of this feature.
 
 
-<h3 id="trigger-scope">
-Trigger Scope/Resolution</h3>
+<h3 id="trigger-name-scoping">
+Trigger Name Scoping</h3>
 
-All [=triggers=] are document-global by default,
-similar to <l spec="css-anchor-position-1">[=anchor names=]</l>.
+A <dfn export>trigger-instantiating property</dfn> specifies the details for
+instantiating an [=Animation Trigger=] along with a name for the instantiated
+trigger, a <dfn export>trigger name</dfn>.
+
+All [=trigger names=] are document-global by default,
+similar to <l spec="css-anchor-position-1">[=anchor names=]</l>, but may be
+scoped by [[#trigger-scope| trigger-scope]].
```

Comment:
This line is confusing, anchor names can also be scoped. I think simplest would be to just remove it. If there is any cross referencing, I could see the anchor scope spec calling out how unlike timeline-scope and trigger-scope it has additional restrictions.

Reply:
Sounds good. I kept the "document-global" part as I don't think that's called out elsewhere... or is that covered by the "tree-scoped" references?